packages/front/svelte/views/user/profile/Profile.svelte

```diff
@@ -0,0 +1,24 @@
+<script>
+    import { ProfileController } from "./Profile.controller";
+    import Logout from "./logout/DeleteUser.svelte";
+    import Button from "@dsfr/Button.svelte";
+
+    const controller = new ProfileController();
+</script>
+
+<div class="fr-container">
+    <div class="fr-grid-row">
+        <div class="fr-col-8">
+            <Logout on:delete-user={() => controller.deleteUser()} />
+        </div>
+    </div>
+    <div class="fr-grid-row fr-mt-8v">
+        <div class="fr-col-3">
+            <Button type="secondary" on:click={controller.logout}>Se déconnecter</Button>
```

Comment:
/!\ Tu risque de perdre de this dans ton logout

Reply:
:D ça c'est jouée avec le feu pour le prochain dev qui va faire un truc dans la methode et passer 30min à comprendre pourquoi y a pas de this :)

Reply:
ok je modifie alors